<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PauseMenu</class>
 <widget class="QMainWindow" name="PauseMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PauseMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="doneButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#doneButton:hover{background-color: rgba(0,0,0,0.5);}
QPushButton#doneButton{background-color:rgba(0,0,0,0); border-radius:15px;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#restartButton:hover{background-color: rgba(0,0,0,0.5);}
QPushButton#restartButton{background-color:rgba(0,0,0,0); border-radius:15px;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#quitButton:hover{background-color: rgba(0,0,0,0.5);}
QPushButton#quitButton{background-color:rgba(0,0,0,0); border-radius:15px;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
